vllm/model_executor/layers/fused_moe/fused_moe.py

```diff
@@ -541,6 +541,236 @@ def fused_moe_kernel(
     tl.store(c_ptrs, accumulator, mask=c_mask)
 
 
+@triton.jit
+def fused_moe_kernel_small_batch_size(
```

Comment:
let's try to consolidate this into the `fused_moe_kernel` to reduce duplicate code